src/runaqa.ts

```diff
@@ -110,27 +76,52 @@ export async function runaqaTest(
   process.chdir('TKG')
   try {
     await exec.exec('make compile')
+
     if (target.includes('custom') && customTarget !== '') {
       const customOption = `${target
         .substr(1)
         .toUpperCase()}_TARGET=${customTarget}`
       await exec.exec('make', [`${target}`, `${customOption}`], options)
-    } else {
+    }
+    else if (target.includes('-f parallelList.mk')) {
+      // move the parallelList to TKG/
+      if (IS_WINDOWS) {
```

Comment:
As all test environment setup is moving to setupTestEnv() I would suggest moving this logic to setupTestEnv().